community/artifactory-ha/templates/nginx-deployment.yaml

```diff
@@ -24,16 +24,14 @@ spec:
         app: {{ template "artifactory-ha.name" . }}
         component: {{ .Values.nginx.name }}
         release: {{ .Release.Name }}
-        heritage: {{ .Release.Service }}
-        chart: {{ template "artifactory-ha.chart" . }}
```

Comment:
Please add `heritage` and `chart` values back unless there is a compelling reason to remove them.